addon/mixins/dom.js

```diff
@@ -110,43 +110,53 @@ export default Mixin.create({
    ```
 
    @method addEventListener
-   @param { String } selector the jQuery selector or element
+   @param { String } selector the DOM selector or element
    @param { String } _eventName the event name to listen for
    @param { Function } _callback the callback to run for that event
    @public
    */
   addEventListener(selector, eventName, callback, _options) {
-    assert('Must provide an element (not a jQuery selector) when using addEventListener in a tagless component.', this.tagName !== '' || typeof selector !== 'string');
+    assert('Must provide an element (not a DOM selector) when using addEventListener in a tagless component.', this.tagName !== '' || typeof selector !== 'string');
     assert('Called addEventListener before the component was rendered', this._currentState === this._states.inDOM);
 
     // Ember.assign would be better here, but Ember < 2.5 doesn't have that :(
     let options = merge(merge({}, DEFAULT_LISTENER_OPTIONS), _options);
     let element = findElement(this.element, selector);
+    let eventNames = eventName.split(' ');
```

Comment:
I think we should drop this with the jquery removal. Specifying the event like `oninput change` is a very "jquery" way of doing it. If we intend to be a thin shim over native API's we should remove this.